pkg/controller/upgradeconfig/upgradeconfig_controller_test.go

```diff
@@ -160,16 +160,15 @@ var _ = Describe("UpgradeConfigController", func() {
 					It("Adds it successfully", func() {
 						matcher := testStructs.NewUpgradeConfigMatcher()
 						util.ExpectGetClusterVersion(mockKubeClient, clusterVersionList, nil)
-						gomock.InOrder(
```

Comment:
`InOrder` didn't play nice with multiple `mockKubeClient.EXPECT().Status()` calls.
Happy to correct this if anyone knows how to do so correctly

Reply:
@sedroche can you please review this as removing `InOrder` weakens time based assertions.